docs/source/quantization_weight_only.md

```diff
@@ -73,7 +73,7 @@ q_model = quantization.fit(model, conf, eval_func=eval_func)
 q_model.save('saved_results')
 ```
 
-The saved_results folder contains two files: `best_model.pt` and `weight_config.json`, and the generated q_model is a fake quantized model.
+The saved_results folder contains two files: `best_model.pt` and `qweight_config.json`, and the generated q_model is a fake quantized model.
```

Comment:
is qweight.config (in json) better? how about pt q config file?

Reply:
we can wait for a proposal from outside when this feature becomes popular.